src/MigrationTools.Clients.AzureDevops.ObjectModel/Processors/Pipelines/ReleaseDefinition.cs

```diff
@@ -0,0 +1,596 @@
+﻿using System;
+using System.Dynamic;
+using Newtonsoft.Json;
+
+namespace MigrationTools.Enrichers.Pipelines
+{
+    public partial class ReleaseDefinition
```

Comment:
Does ReleaseDefenition belong in "DataContracts"?

Reply:
It does! I already made som changes to `ReleaseDefinition` and `BuildDefinition`. Since they have multiple parameters in common, i created an abstract class, from which they inherit